playbooks/de-populate-consul.yaml

```diff
@@ -93,3 +93,51 @@
         - {key: "configs/{{environment_name}}/notification-agent/clean/age", value: "{{ notificationagent.clean_age }}"}
         - {key: "configs/{{environment_name}}/notification-agent/clean/enable", value: "{{ notificationagent.clean_enable }}"}
         - {key: "configs/{{environment_name}}/services/port", value: "{{ default_service_port }}"}
+        - {key: "configs/{{environment_name}}/de/default-build-number", value: "{{ de_version_name }}"}
+        - {key: "configs/{{environment_name}}/de/release-version", value: "{{ de_version }}"}
+        - {key: "configs/{{environment_name}}/de/base-url", value: "{{ cas_base }}"}
+        - {key: "configs/{{environment_name}}/de/cas-server-name", value: "{{ de.app_base }}"}
+        - {key: "configs/{{environment_name}}/de/cas-validation", value: "/iplant-cas-ticket-validator"}
+        - {key: "configs/{{environment_name}}/de/cas-logout-url", value: "/iplant-cas-logout"}
+        - {key: "configs/{{environment_name}}/de/cas-app-name", value: "{{ de.description }}"}
+        - {key: "configs/{{environment_name}}/de/cas-login-url", value: "/login"}
+        - {key: "configs/{{environment_name}}/de/authorized-groups", value: "{{ admin_groups }}"}
+        - {key: "configs/{{environment_name}}/de/group-attribute-name", value: "{{ cas_group_attribute }}"}
+        - {key: "configs/{{environment_name}}/de/cas-no-logout-url", value: "{{ cas_no_logout_url }}"}
+        - {key: "configs/{{environment_name}}/de/cas-app-list", value: "{{ cas_app_list }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-service", value: "{{ de.app_base }}/de/{{ de.rpc_name }}/{{ de.empty_url }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-target", value: "{{ cas_base }}/login?service={{ de.app_base }}/de/{{ de.rpc_name }}/{{ de.empty_url }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-interval", value: "90"}
+        - {key: "configs/{{environment_name}}/de/jwt-private-key-path", value: "{{ jwt.signing_key.private }}"}
```

Comment:
Hm. It looks like these JWT properties will be used by terrain as well. I think it'd be cleaner to do this as `/jwt/private-key-path` (and parallel/same idea for password) so both things can pull from the same place easily/nicely.